testing/pypy3/libpthread_version.patch

```diff
@@ -0,0 +1,13 @@
+diff -r b30914a448be pypy/module/sys/system.py
```

Comment:
Why is this patch needed (please add patch header with description)? Have you sent it also to upstream, or is it specific for Alpine?